build/templates/enums.py.mako

```diff
@@ -10,6 +10,9 @@ from enum import Enum
 
 
 class ${enums[enum_name]['python_name']}(Enum):
+<%
+print_list = []
+%>\
```

Comment:
Can this be indented one more level, so that it matches the code below?

Reply:
No, any space added will show in the generated files. The best I can do is to indent the python code.

```
<%
    first = False
%>\
```

Reply:
> The best I can do is to indent the python code.

That sounds good.